### DISCUSSION TITLE
Long review process for extensions

### DISCUSSION THREAD

**Discussion by NC22** [microsoft/MicrosoftEdge-Extensions#114] [Q&A] [answered]
Hello. I develop few extensions, and published them in extension store while ago. 

Extension that under spam attack :

https://microsoftedge.microsoft.com/addons/detail/kellyc-show-youtube-disli/kdjonhbaoaghhppmglnhaigpabhnmhgi

This ext requires recent updates when parent site is changes its layout, to keep work properly.

When this extensions was not very popular it wasnt a big deal long review process like 8+ days and more. But now when extension keeped in review state longer then few days, that ruins user experience during review time and many users start spam "1 start reviews" on extension page without any instruments to protect extension page from that during publish \ audit process.

How to deal with this problem? I don't mind if review process need take some time, but why people allowed to make "1 star" reviews while this process is continuing - why there is no any information on extension page that new version already uploaded and under review, to let people know that developer is know about issue and already sended new version.

Also currently I can't do anything with this reviews from my control panel - for some reason there is no any "report spam" button, I can only reply on each one review separately without any possibility to add information on extension page. Also my reply will see only user that send this review, because review section depends on user country ( not only by language), so there is no common review list page for English language and they all separated, and I cant figure out where placed each one of reviews to report them.

Where I can report this problem, and who can help with clean up page from spam attack? Currently this attack continue https://catface.ru/userfiles/media/udata_1697507735_dslmxbft.png but actual new version was uploaded already long time ago - 13.10.23 and still no any updates \ audit feedback.

**Comment by joelataccelm** [comment]
Inform your customers Microsoft takes 10 days to approve an update, and insist they switch to safari, firefox or chrome. Publicly shame Microsoft for being completely useless.

**Comment by ManikanthMSFT** [comment]
Hi @NC22 , @joelataccelm, thank you for your feedback. We sincerely apologize for the inconvenience caused by the delay in the approval process. Please be assured that we have taken your feedback seriously, and our team is actively working to reduce the review time to make the process faster.

**Comment by joelataccelm** [comment]
@ManikanthMSFT I love the sound of that.  
If you can help me out today, I would greatly appreciate it. extension aepambcpolknomodpkihfmocblnjfelm has version 3.72 awaiting review, but it contains a critical fix that needs to be up before tonight.

**Comment by venkatasai5171** [comment, marked as answer]
Hello,

I’m Venkat from the Microsoft Edge Extension Developer Support Team.

Currently, the review process typically takes 5 to 7 business days to complete.

If you’d like to report comments or feedback, you can do so directly in the Edge Add-ons Store. A dedicated button is provided for this purpose to make the process easier.

Thank you for your cooperation and understanding.

Regards,
Venkat
